Doxyfile

```diff
@@ -733,7 +733,7 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ./hazelcast/
+INPUT                  = ./hazelcast/ README.md
 
```

Comment:
There's a space left of README.md

Reply:
they need to be separated by a space. The first one is a folder, the second one is a file name.